### PR TITLE
Add south for database migrations

### LINKED ISSUE #125: 'no such column: dashboard_announcement.expiration'
Introduced by #83. Need to migrate database after adding new field to announcement. I think we got south working for this, but I don't know what the command is.

### PR DESCRIPTION
**Pull request by natmote**
Also adds migration files, and a script that gets your database up to date
and using south.

After updating, please run the `conver_to_south.sh` script in the root of the repository (after, of course, running `pip install -r requirements.txt` to add south). This script will apply the necessary modifications to your database to get south working, and it will also migrate your database to add the column that I tried to add in #123.

This resolves #125.

### CODE REVIEW

File: fullhouse/dashboard/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            'invite_key': ('django.db.models.fields.CharField', [], {'max_length': '40'}),
+            'sent_date': ('django.db.models.fields.DateTimeField', [], {})
+        },
+        'dashboard.userprofile': {
+            'Meta': {'object_name': 'UserProfile'},
+            'birthday': ('django.db.models.fields.DateField', [], {'null': 'True'}),
+            'house': ('django.db.models.fields.related.ForeignKey', [], {'related_name': "'members'", 'null': 'True', 'on_delete': 'models.SET_NULL', 'to': "orm['dashboard.House']"}),
+            'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'}),
+            'user': ('django.db.models.fields.related.OneToOneField', [], {'related_name': "'profile'", 'unique': 'True', 'to': "orm['auth.User']"})
+        }
```
Comment (j3ffml) [inline fullhouse/dashboard/migrations/0001_initial.py:128]: Are these autogenerated, or do they have to be written by hand? Seems like we are duplicating a lot of information.

**Comment by j3ffml** [comment]
The 0001 and 0002 files look autogenerated. Did you mean to add them?

File: fullhouse/dashboard/migrations/0001_initial.py (hunk 1 above)
Comment (natmote) [inline fullhouse/dashboard/migrations/0001_initial.py:128]: They are auto generated

**Comment by natmote** [comment]
They are auto generated, but this is how south works. I did mean to add them.

**Comment by j3ffml** [comment]
So they will change everytime we do a migration then?

Edit: looked it up, I get it now. lgtm

**Comment by natmote** [comment]
no, there will be another added every time we do a migration.

**Pull request merged**